registry/index/templates/about.html

```diff
@@ -10,9 +14,56 @@ <h1>About the HT Phenotyping Project</h1>
 
                 <h2>The HT Phenotyping Transfer Service</h2>
 
-                Data transfers from data sources to storage and processing
-                servers are facilitated by
-                <a href="https://research.cs.wisc.edu/htcondor/">HTCondor</a>.
+                <p>
+                    Data transfers from data sources to storage and processing
+                    servers are facilitated by
+                    <a href="https://research.cs.wisc.edu/htcondor/">HTCondor</a>.
+                    The HT Phenotyping project uses HTCondor to transfer data
+                    in an automated, reliable, and secure manner.
+                </p>
+
+                <h3 id="transfer-service-requirements">Requirements for using the Transfer Service</h3>
+
+                <p>
+                    To use the transfer service, you must have a
+                    properly set-up computer to run the "client":
+                </p>
+
+                <ol>
+                    <li>
+                        You must have a computer running the
+                        <a href="https://releases.ubuntu.com/18.04.4/">Ubuntu 18</a>
+                        operating system.
+                    </li>
+                    <li>
+                        You must have administrative privileges on the computer.
+                        For Ubuntu, this means you can run the
+                        <code>sudo</code> command.
+                    </li>
+                    <li>
+                        The computer should be powered on at all times
```

Comment:
I think this can be worrying for people who have a computer out at a field... the sync will still work even if the computer isn't powered on right at midnight. Maybe something like:

The computer should be powered on overnight if possible as data sync is scheduled to occur at midnight or as soon as is possible thereafter.

Reply:
Revised the requirement. I don't want to get too much into how the resiliency works... but I might be wrong about that.

Reply:
Agree, hard to be precise without being wordy. What you have is good enough.